docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -47,9 +50,9 @@ Qute.fmt("I am {#if ok}happy{#else}sad{/if}!", Map.of("ok", true)); <5>
 <3> A builder-like API is available for more complex formatting requirements.
 <4> Note that for a "text/html" template the special chars are replaced with html entities by default.
 <5> You can use any xref:basic-building-blocks[building block] in the template.
-In this case, the xref:if_section[] is used to render the appropriate part of the message based on the input data.
+In this case, the xref:if_section[If section] is used to render the appropriate part of the message based on the input data.
```

Comment:
@MichalMaler: Just a question, should "section" be included as part of the xref name here? That is, is the name of the section "If" rather than "If section"?

Reply:
Hello Sheila. Hope you are doing good.
The same file contains a section called "If Section". Adding "If section" description manually because thats the best way in my opinion how to call that chapter and its name, while taking in a consideration that headings in this doc don't follow the proper capitalization, and was just adding what appers there in docs when displayed on a portal. 
HOWEVER! :) Maybe a better context would be "if condition" instead of "if section", but I am not the author.

With this fix, I just apply what was displayed there anyway. And it is a good example of the fact that even anchors with [] empty brackets sometimes renders out of the context meanings, so even those should be checked from time to time. 
CC @maxandersen